### DISCUSSION TITLE
Expert mode thoughts and ideas

### DISCUSSION THREAD

**Discussion by fulwerin** [NillerMedDild/Enigmatica6#2900] [Ideas]
I have now spent a few hours in Expert mode. What strikes me is that almost every Expert modpack always follows two strands of technology and magic. I would welcome it once that would be mixed. Also, in the 1.16 comes to explore which has become a very important point.

My idea; Why not a Bloodmagic altar in the Dissolution Chamber craften or a chip with the Natures Altar. Also it would be possible that certain mobs drop items such as the Undergarden mobs. Of course these are only small ideas but I want to symbolize the mix of technology and magic. 

Why do I see a problem is that most of the time technique is played first and only magic is used to fly or because there is a good item for the automations, why not the other way around? first you need mana for a storage system.

Thanks for reading and thanks for the great modpack.

**Comment by NielsPilgaard** [comment]
Thanks for the thoughts, I took the liberty of converting your issue to a discussion, I hope you don't mind.

We're trying to intertwine magic and tech a bit, but I think it's important not to overdo it.

**Reply by fulwerin** [reply]
Yes is not a problem.
Not overdoing it is certainly good. I simply miss the connection between technology and magic at the beginning. It would also go simple things like a chest in the mana pool or sticks with the Occultism fire.
